autoPyTorch/pipeline/components/setup/network/backbone/image.py

```diff
@@ -0,0 +1,229 @@
+from collections import OrderedDict
+from typing import Any, Dict, Optional, Tuple, List
+
+import ConfigSpace as CS
+import torch
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.hyperparameters import (
+    CategoricalHyperparameter,
+    UniformFloatHyperparameter,
+    UniformIntegerHyperparameter
+)
+from torch import nn
+from torch.nn import functional as F
+
+from autoPyTorch.pipeline.components.setup.network.backbone.base_backbone import BaseBackbone
+
+
+class ConvNetBackbone(BaseBackbone):
```

Comment:
There can also be a ConvNetBackbone for e.g. time series. So we might want to consider renaming this to something like `ConvNetImageBackbone` or `ConvNet2DBackbone` and creating an additional `ConvNetBackbone` that acts as a parent class for `ConvNetImageBackbone` and `ConvNetTimeSeriesBackbone`/`ConvNet1DBackbone` (of course only if there actually is something to share). The same goes for other network types that can be used for different modalities but only if it makes

Reply:
Will rename it and will add the `ConvNetTimeSeriesBackbone` to the issue about the networks we want to implement

Reply:
se #32